docs/getting_started.md

```diff
@@ -180,9 +180,9 @@ argocd app sync guestbook
 
 This command retrieves the manifests from the repository and performs a `kubectl apply` of the
 manifests. The guestbook app is now running and you can now view its resource components, logs,
-events, and assessed health status:
+events, and assessed health status.
 
-### From UI:
+### Synching via UI
```

Comment:
There's a small typo, should be `Syncing` I guess

Reply:
y'know I googled this too because I thought it looked weird, but google told me it was correct. I guess both are technically correct, but I do like syncing much better. Thanks for catching that, I'll fix it right away